book/src/passive-stake-delegation-and-rewards.md

```diff
@@ -0,0 +1,149 @@
+# Stake Delegation and Reward
+
+This design proposal focuses on the software architecture for the on-chain
+voting and staking programs.  Incentives for staking is covered in [staking
+rewardsd](staking-rewards.md).
+
+The current architecture requires an active vote for each delegated stake from
+the validator, and therefore does not scale to allow for replicator clients
+automatically delegate their rewards.
+ 
+The design proposes a new set of programs for voting and stake delegation, The
+proposed programs allow many stakes to passively earn rewards with a single
```

Comment:
stakes -> stakers?

Reply:
i think `stakes` is correct.  A staker is a user, and can have many stakes.

Reply:
If not comfortable with stakers, how about stake accounts?